tests/lib/rules/require-macro-variable-name.js

```diff
@@ -0,0 +1,296 @@
+      </script>
+      `,
+      errors: [
+        {
+          message: 'The variable name of "useSlots" must be "slots".',
+          line: 3,
+          column: 15,
+          suggestions: [
+            {
+              desc: 'Change the variable name to "slots".',
+              output: `
+      <script setup>
+        const slots = useSlots({})
+      </script>
+      `
+            }
+          ]
+        }
+      ]
+    },
+    {
+      filename: 'test.vue',
+      code: `
+      <script setup>
+        const propsWrong = withDefaults(defineProps(['foo', 'bar']), {
+          foo: 42,
+          bar: 'abc'
+        })
+      </script>
+      `,
+      errors: [
+        {
+          message: 'The variable name of "defineProps" must be "props".',
+          line: 3,
+          column: 15,
+          suggestions: [
+            {
+              desc: 'Change the variable name to "props".',
+              output: `
+      <script setup>
+        const props = withDefaults(defineProps(['foo', 'bar']), {
+          foo: 42,
+          bar: 'abc'
+        })
+      </script>
+      `
+            }
+          ]
+        }
+      ]
+    },
+    {
+      filename: 'test.vue',
+      code: `
+      <script setup>
+        const slots = defineSlots({})
+        const useSlots = useSlots({})
+        const attrs = useAttrs({})
+      </script>
+      `,
+      errors: [
+        {
+          message: `The variable name of "defineSlots" must be "${customOptions.defineSlots}".`,
+          line: 3,
+          column: 15,
+          suggestions: [
+            {
+              desc: `Change the variable name to "${customOptions.defineSlots}".`,
+              output: `
+      <script setup>
+        const ${customOptions.defineSlots} = defineSlots({})
+        const useSlots = useSlots({})
+        const attrs = useAttrs({})
+      </script>
+      `
+            }
+          ]
+        },
+        {
+          message: `The variable name of "useSlots" must be "${customOptions.useSlots}".`,
+          line: 4,
+          column: 15,
+          suggestions: [
+            {
+              desc: `Change the variable name to "${customOptions.useSlots}".`,
+              output: `
+      <script setup>
+        const slots = defineSlots({})
+        const ${customOptions.useSlots} = useSlots({})
+        const attrs = useAttrs({})
+      </script>
+      `
+            }
+          ]
+        },
+        {
+          message: `The variable name of "useAttrs" must be "${customOptions.useAttrs}".`,
+          line: 5,
+          column: 15,
+          suggestions: [
+            {
+              desc: `Change the variable name to "${customOptions.useAttrs}".`,
+              output: `
+      <script setup>
+        const slots = defineSlots({})
+        const useSlots = useSlots({})
+        const ${customOptions.useAttrs} = useAttrs({})
+      </script>
+      `
+            }
+          ]
+        }
+      ],
+      options: [customOptions]
+    }
```

Comment:
Could you please add a test case for when omit some of the options?
For example, if we use an option other than `useAttrs`, what is the `useAttrs` variable name forced to?